test/helpers/test.go

```diff
@@ -103,6 +104,34 @@ func (t *TestOptions) GetGitOrganisation() string {
 	return org
 }
 
+func (t *TestOptions) GetGitProvider() (gits.GitProvider, error) {
+	authConfigService, err := auth.NewFileAuthConfigService(fmt.Sprintf("%s/.jx/gitAuth.yaml", os.Getenv("HOME")))
```

Comment:
Should we really be using `~/.jx/gitauth.yaml` and not be getting the credentials from the test cluster instead?

Reply:
Probably not -- I wasn't sure how to tackle this. I didn't see an easy way to create an auth service any other way.

Reply:
@pmuir It looks like `jx step git credentials` just uses a kube client to pull secrets and write to a file. Would the canonical thing be to use that or to just pull the secrets directly?

Reply:
After consulting with @abayer the current solution is to look in `JX_HOME` first, which applies to running the tests in the test clusters, and then `HOME` if it's unset, which applies to running it locally.